components/01-atoms/typography/headings/_headings.scss

```diff
@@ -34,4 +34,14 @@ h6 {
   h6 {
     @include tokens.h6-yale-new;
   }
+
+  * + {
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      margin-top: 1em;
```

Comment:
Should this be `var(--size-spacing-7)` or another token ?

Reply:
This might be better tackled as a mixin instead of re-declaring the heading selectors again.

```
@mixin header-spacing {
   &:not(:first-child) {
      margin-top: var(--size-spacing-7);
   }
}
```

Reply:
The mixin works for me.

(same comment as below about the value though, it's `em`, nor `rem` so there's no token for that.)